components/builder-api-client/src/lib.rs

```diff
@@ -301,6 +323,24 @@ impl Tabular for PendingOriginInvitationsResponse {
     }
 }
 
+impl TabularText for OriginInfoResponse {
+    fn as_tabbed(&self) -> Result<String> {
+        let tw = tabw().padding(2).minwidth(5);
+        let mut body = Vec::new();
+        body.push(String::from("Owner Id\tOwner Account\tPrivate Key\tPackage Visibility"));
+        body.push(format!("{}\t{}\t{}\t{}",
+                          self.owner_id,
+                          self.owner_account,
+                          self.private_key_name,
+                          self.default_package_visibility));
+        tabify(tw, &body.join("\n"))
+    }
+}
+
+impl PortableText for OriginInfoResponse {
```

Comment:
Having `json` output is awesome! Far outside the scope of this PR, and just thinking outloud. It would be awesome to have all CLI output be represented as structs that implements `Serialize`. We could then easily support all kinds of output. We could even implement (or maybe one exists) a `Tabular` format for `serde`.

Reply:
That's a fantastic idea!